src/v2/Components/ArtworkFilter/ArtworkFilterArtworkGrid.tsx

```diff
@@ -70,7 +78,17 @@ const ArtworkFilterArtworkGrid: React.FC<ArtworkFilterArtworkGridProps> = props
           itemMargin={40}
           user={user}
           onClearFilters={context.resetFilters}
-          emptyStateComponent={context.ZeroState && <context.ZeroState />}
+          emptyStateComponent={
+            isArtistPage ? (
+              !!hasAppliedFilters ? (
+                <ArtworkGridEmptyState onClearFilters={context.resetFilters} />
+              ) : (
+                <ZeroState />
+              )
```

Comment:
**suggestion**: what do you think about passing `ZeroState` props in `src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx`? Something like this

```javascript
<ArtworkFilterContextProvider
  aggregations={aggregations}
  counts={artist.counts as Counts}
  filters={filters}
  sortOptions={[...]}
  ZeroState={ZeroState} // Pass component here
>
  <BaseArtworkFilter
    relay={relay}
    viewer={artist}
    relayVariables={{
      aggregations: ["TOTAL"],
    }}
    enableCreateAlert
    savedSearchEntity={savedSearchEntity}
  />
</ArtworkFilterContextProvider>
```

Reply:
that's a much cleaner solution thanks!